F/Flowstar/bundled/patches/windows_mkdir.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/include.h b/include.h
+index e246927..ff3e8ad 100644
+--- a/include.h
++++ b/include.h
+@@ -31,6 +31,11 @@
+ #include <gsl/gsl_linalg.h>
+ #include <glpk.h>
+ 
++#ifdef _WIN32
+#include <direct.h>
```

Comment:
Remind me not to manually edit patch files :roll_eyes: 
```suggestion
+#include <direct.h>
```

Reply:
> Remind me not to manually edit patch files 🙄

😆  Does that earn a YOLO badge? 

Thanks for all you help and patience. I'm a bit out of my element here